Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "incident_createdby",
+    "id": "indicator_cyrenmalwarefamily",
```

Comment:
Why are you changing the incident field's id?

Reply:
I was getting the following errors, so I tried to fix them: 

Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json: [ST102] - The file id has changed from indicator_cyrenmalwarefamily to incident_createdby

Packs/Cyberint/IncidentFields/incidentfield-Cyberint_Created_By.json: [ST103] - You've added fromversion to an existing file in the system, this is not allowed, please undo.